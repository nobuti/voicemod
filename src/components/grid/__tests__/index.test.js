import React from "react";
import { render, screen } from "@testing-library/react";
import Grid from "../";

describe("Grid", () => {
  test("should render properly", () => {
    const { container } = render(
      <Grid title="Wadus">
        <li>wadus</li>
        <li>wadus</li>
      </Grid>
    );

    expect(container).toMatchSnapshot();
    expect(screen.getByTestId("grid")).toBeInTheDocument();
    expect(screen.getByTestId("grid-title").textContent).toEqual("Wadus");
  });
});
