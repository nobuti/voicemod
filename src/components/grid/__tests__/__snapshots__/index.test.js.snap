// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should render properly 1`] = `
<div>
  <section
    class="Section"
    data-testid="grid"
  >
    <h1
      class="Heading"
      data-testid="grid-title"
    >
      Wadus
    </h1>
    <ul
      class="Grid"
      data-testid="grid-list"
    >
      <li>
        wadus
      </li>
      <li>
        wadus
      </li>
    </ul>
  </section>
</div>
`;
